Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -35,7 +35,17 @@ public class Interactable :
         /// Setup the input system
         /// </summary>
         private static IMixedRealityInputSystem inputSystem = null;
-        protected static IMixedRealityInputSystem InputSystem => inputSystem ?? (inputSystem = MixedRealityToolkit.Instance.GetService<IMixedRealityInputSystem>());
+        protected static IMixedRealityInputSystem InputSystem
+        {
+            get
+            {
+                if (InputSystem == null)
```

Comment:
This seems like it'll call itself recursively forever? Happens in a few scripts in this PR.

```suggestion
                if (inputSystem == null)
```

Reply:
thanks! missed that